src/plugins/team-goal-mixin.js

```diff
@@ -48,10 +48,17 @@ export default {
 		daysLeft() {
 			const start = this.goal?.startDate ? new Date(this.goal?.startDate) : new Date();
 			const end = this.goal?.endDate ? new Date(this.goal?.endDate) : new Date();
-			return intervalToDuration({
+			const interval = intervalToDuration({
```

Comment:
I think we may want to use one of the "days" based methods instead. Maybe [differenceInDays](https://date-fns.org/v3.6.0/docs/differenceInDays)

Reply:
That said, there is still opportunity with both your current method and the new one to have an offset in hours due to timezones when using dates from the goal service which are static...